### PR TITLE
uh can i edit 2 strings and make a pull request?

### PR DESCRIPTION
**Pull request by shogunator1337**
Thanks!!
heap is love heap is life

### CODE REVIEW

File: server/client_manager.py [hunk 1]
```diff
@@ -290,23 +294,17 @@ def disemvowel_message(self, message):
...
-        c = self.Client(self.server, transport, cur_id, self.server.get_ipid(transport.get_extra_info('peername')[0]))
+        c = self.Client(self.server, transport, heappop(self.cur_id), self.server.get_ipid(transport.get_extra_info('peername')[0]))
         self.clients.add(c)
-        self.cur_id[cur_id] = True
         return c
 
             
     def remove_client(self, client):
-        self.cur_id[client.id] = False
+        heappush(self.cur_id, client.id)
```
Comment (oldmud0) [inline server/client_manager.py:306]: Good thinking. It took me a while to understand what was going on here - basically get the smallest unused id in optimal time.

File: server/client_manager.py [hunk 2]
```diff
@@ -204,13 +205,16 @@ def send_area_info(self, area_id, mods):
             info = ''
             if area_id == -1:
                 # all areas info
-                info = '== Area List =='
+                cnt = 0
+                info = '\n== Area List =='.format(len(self.server.area_manager.areas))
```
Comment (oldmud0) [inline server/client_manager.py]: There don't seem to be any replacement fields in this string. The format will have no effect.

File: server/client_manager.py [hunk 3]
```diff
@@ -247,7 +251,7 @@ def send_done(self):
             self.send_command('BN', self.area.background)
             self.send_command('LE', *self.area.get_evidence_list(self))
             self.send_command('MM', 1)
-            self.send_command('OPPASS', fantacrypt.fanta_encrypt(self.server.config['guardpass']))
+            self.send_command('OPPASS', fantacrypt.fanta_encrypt(''))
```
Comment (oldmud0) [inline server/client_manager.py]: I'm guessing the OPPASS command was a useless feature added by Fanat 😕

File: config_sample/config.yaml [hunk 4]
```diff
@@ -3,11 +3,9 @@ playerlimit: 100
...
 local: false
 
-guardpass: guard
 modpass: mod
 
 motd: Welcome to my server!
-help_msg: TODO
 
 use_district: false
 district_ip: 127.0.0.1
```
Comment (oldmud0) [inline config_sample/config.yaml:11]: Next to nuke: probably districts.

**Review by oldmud0 — APPROVED** [review]

File: config_sample/config.yaml (hunk 4 above)
Comment (shogunator1337) [reply config_sample/config.yaml:11]: but district WORKS

**Pull request merged**